src/meta/src/manager/env.rs

```diff
@@ -411,16 +432,31 @@ impl MetaSrvEnv {
                 }
             }
             MetaStoreImpl::Sql(sql_meta_store) => {
+                let mut is_sql_backend_cluster_first_launch = true;
+                if let MetaStoreImpl::Sql(sql_store) = &meta_store_impl {
+                    is_sql_backend_cluster_first_launch =
+                        is_first_launch_for_sql_backend_cluster(sql_store).await?;
+                    // Try to upgrade if any new model changes are added.
+                    Migrator::up(&sql_store.conn, None)
```

Comment:
In L376, `meta_store_impl` is passed to new `NotificationManager`. Have we checked that whether it is correct to init `NotificationManager` before `Migrator::up` ?